contracts/libraries/MultiSend.sol

```diff
@@ -61,9 +62,10 @@ contract MultiSend {
                 case 1 {
                     success := delegatecall(gas(), to, data, dataLength, 0, 0)
                 }
-                if eq(success, 0) {
```

Comment:
I believe the reason we didn't do memory-safe assembly here is the same as the `SafeProxy` contract, it gets compiled in its unit. Why do we need to make it memory-safe? (not opposed to the change, just wondering)

Reply:
> it gets compiled in its unit.

It does, but if you use Foundry and deploy `MultiSend` with other Safe contracts in a script (how I found the issue), then it gets it would get included in the same compilation unit. This is different from the `SafeProxy` contract, since it gets compiled in a separate unit first for it to be used in the `SafeProxyFactory` implementation.